realm/realm-library/src/main/cpp/io_realm_SyncManager.cpp

```diff
@@ -41,34 +41,35 @@ using namespace realm::jni_util;
 
 std::unique_ptr<Client> sync_client;
 
-JNIEXPORT void JNICALL Java_io_realm_SyncManager_nativeInitializeSyncClient
-    (JNIEnv *env, jclass)
+JNIEXPORT void JNICALL Java_io_realm_SyncManager_nativeInitializeSyncClient(JNIEnv* env, jclass)
 {
     TR_ENTER()
-    if (sync_client) return;
+    if (sync_client)
+        return;
```

Comment:
Style issue for core: https://github.com/realm/realm-core/issues/2294. Library and core should have the same fixed/uniform style.